README.md

```diff
@@ -3,7 +3,7 @@ The only purpose of this repo is to be used as technical test for new QA Enginee
 
 
 ## What are the requirements to setup this project?
-This project requires `NodeJS` intalled on version 18+.
+This project requires `Docker Desktop` intalled.
```

Comment:
This project requires `NodeJS`, on version 18, or `Docker` installed. Depending on the way you prefer to run it.